tools/ddsrouter_tool/src/cpp/main.cpp

```diff
@@ -65,6 +68,13 @@ int main(
         return static_cast<int>(arg_parse_result);
     }
 
+    if (print_version)
+    {
+        std::cout << "DDSRouter v" << DDSROUTER_CORE_VERSION_STR << ", branch: " << DDSROUTER_CORE_BRANCH <<
```

Comment:
I would prefer to have the print in a separated method, so it is easier to find and modify

Reply:
NIT: this is totally debatable, but we do prefer to define the functions in the `.hpp` rather than implement them in `.cpp`, even when it will not be used anywhere else.
This is because:
- It avoids problems of sorting lines of code
- It is easier to find from a developer that navigates the code (we usually search in .hpp rather than see the actual code to understand a file or a module)